### ISSUE TITLE
adjust test test_unprivileged_user_odf_access

### ISSUE THREAD

**Issue by DanielOsypenko** [red-hat-storage/ocs-ci#8098]
Failing on login page, can't find locator (login_loc["skip_tour"])

```
if username is not constants.KUBEADMIN:
        element = wait.until(ec.element_to_be_clickable((login_loc["skip_tour"])))
        element.click()
    return driver
```

https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/465/12961/593770/594078/594079/log?logParams=history%3D594079%26page.page%3D1

http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/j-073vu1ce33-ua/j-073vu1ce33-ua_20230721T084807/logs/ui_logs_dir_1689939392/screenshots_ui/test_unprivileged_user_odf_access/2023-07-21T21-37-33.958169.png

**Comment by DanielOsypenko** [comment]
adding another failure here. 
https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/465/14002/636066/636369/636370/log

**Comment by DanielOsypenko** [comment]
Fixed at #8393 waits for merging

**Comment by ebondare** [comment]
Fixed by https://github.com/red-hat-storage/ocs-ci/pull/8393
